src/assets/style/ShortcutGuideStyle.css

```diff
@@ -4,101 +4,138 @@
-.a11y_modal {
-	display: none; /* Hidden by default */
-	position: fixed; /* Stay in place */
-	z-index: 1000; /* Placed on top */
+.shortcut-guide__background {
+	display: block;
+	position: fixed;
+	z-index: 1000;
 	left: 0;
 	top: 0;
-	width: 100%; /* Full width */
-	height: 100%; /* Full height */
-	overflow: auto; /* Enable scroll if needed */
-	background-color: rgb(0, 0, 0); /* Fallback color */
-	background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
-	font-family: "Open Sans", sans-serif;
-	font-size: small;
-	color: #f5f5ff;
-}
-
-.a11y_no_scroll {
-	overflow: hidden; /* disable scrolling on the main content */
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgba(0, 15, 58, 0.35);
+	color: #000f3a;
 }
 
-/* Modal Content/Box */
-.a11y_modal_content {
-	border-radius: 25px;
-	margin: 7% auto; /* 15% from the top and centered */
-	padding: 1rem;
-	padding-right: 2rem;
-	padding-left: 2rem;
-	border: 1px solid #888;
-	width: 80%; /* Could be more or less, depending on screen size */
-	border-radius: 21px;
-	background: rgba(0, 0, 0, 0.77);
+.shortcut-guide__container {
+	font-family: "Roboto";
+	font-size: small;
+	margin: 10% auto;
+	padding: 0;
+	border: none;
+	width: 90%;
+	border-radius: 7px 7px;
+	background: #ffffff;
 	backdrop-filter: blur(6.5px);
 }
 
-/* The Close Button */
-#guide_close {
+.shortcut-guide__button-close {
 	color: #aaa;
 	font-size: 25px;
-	background-color: transparent;
-	background-repeat: no-repeat;
+	background: transparent no-repeat;
 	border: none;
 	height: 2rem;
+	cursor: pointer;
+	margin: 0;
 }
 
-#guide_close:hover,
-#guide_close:focus {
+.shortcut-guide__button-close:hover,
+.shortcut-guide__button-close:focus {
 	color: black;
 	text-decoration: none;
-	cursor: pointer;
-}
-
-#guide_close,
-#guide_title,
-#guide_close_sc {
-	margin: 0;
 }
 
-#guide_title {
+.shortcut-guide__title {
 	margin-right: auto;
+	font-size: 16px;
+	font-weight: 700;
 }
 
-#guide_close_sc {
+.shortcut-guide__action-label {
+	display: none;
 	margin-right: 0.5rem;
 	font-weight: lighter;
 }
 
-#guide_container {
+.shortcut-guide__break {
+	display: none;
+}
+
+.shortcut-guide__header {
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
+	background-color: #f5f5f7;
+	padding-left: 16px;
+	padding-right: 16px;
+	border-radius: 7px 7px 0 0;
+	border: none;
 }
 
-.a11y_flex_container {
-	display: flex;
-	flex-direction: row;
-	justify-content: space-evenly;
+.shortcut-guide__cell--empty {
+	display: none;
 }
 
-.a11y_title {
+.shortcut-guide__cell--title,
+.shortcut-guide__cell--explanation {
 	text-align: left;
-	padding-left: 1rem;
+	padding-left: 0;
 }
 
-.a11y_explanation {
-	text-align: left;
-	padding-left: 1rem;
-	font-weight: lighter;
+.shortcut-guide__cell--title,
+.shortcut-guide__cell--shortcut {
+	font-size: 12px;
 }
 
-.a11y_shortcut {
-	text-align: right;
+.shortcut-guide__cell--title {
+	max-width: 100px;
+	white-space: nowrap;
+	font-weight: 700;
```

Comment:
Consider limiting this title to one line only and adding an ellipsis at its end.